examples/price_printer/main.rs

```diff
@@ -1,11 +1,14 @@
 extern crate tycho_simulation;
+#[path = "../shared/utils.rs"]
```

Comment:
aff this is a bit sucky. what about putting this in src/utils.rs ? `load_all_tokens` is there